packages/pangea-node-sdk/src/services/authn/user/authenticators.ts

```diff
@@ -0,0 +1,43 @@
+import PangeaResponse from "@src/response.js";
+import BaseService from "@src/services/base.js";
+import PangeaConfig from "@src/config.js";
+import { AuthN } from "@src/types.js";
+
+export default class UserAuthenticators extends BaseService {
+  constructor(token: string, config: PangeaConfig) {
+    super("authn", token, config);
+  }
+
+  // TODO: Docs
+  /**
+   * @summary Delete MFA Enrollment
+   * @description Delete MFA enrollment for a user.
+   * @operationId authn_post_v1_user_mfa_delete
+   * @param {String} userID - The identity of a user or a service
+   * @param {AuthN.MFAProvider} mfaProvider - Additional mechanism for authenticating
+   * a user's identity
+   * @returns {Promise<PangeaResponse<{}>>} - A promise
+   * representing an async call to the endpoint.
+   * @example
+   * ```js
+   * await authn.user.mfa.delete(
+   *   "pui_zgp532cx6opljeavvllmbi3iwmq72f7f",
+   *   AuthN.MFAProvider.TOTP
+   * );
+   * ```
+   */
+  delete(
+    request:
+      | AuthN.User.Authenticators.Delete.EmailRequest
+      | AuthN.User.Authenticators.Delete.EmailRequest
```

Comment:
What is meant by this:

```
    request:
      | AuthN.User.Authenticators.Delete.EmailRequest
      | AuthN.User.Authenticators.Delete.EmailRequest
```

Why two of the same types?

Reply:
My bad, one should be IDRequest